dlp/snippets/inspect/inspect_bigquery.go

```diff
@@ -43,7 +43,7 @@ func inspectBigquery(w io.Writer, projectID string, infoTypeNames []string, cust
 
 	client, err := dlp.NewClient(ctx)
 	if err != nil {
-		return fmt.Errorf("dlp.NewClient: %w", err)
+		return fmt.Errorf("dlp.NewClient: %v", err)
```

Comment:
Revert. `%w` is the correct formatting directive.

Reply:
fixed it.